packages/docusaurus-plugin-content-docs/src/__tests__/__fixtures__/sidebars/sidebars-category.js

```diff
@@ -9,30 +9,25 @@ module.exports = {
   docs: [
     {
       type: 'category',
-      collapsed: true,
```

Comment:
Removed on purpose, to test that the normalization process actually fallbacks to true